recipes/yac/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "yac" %}
+{% set version = "3.4.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.dkrz.de/dkrz-sw/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: 03d753bae4bc2a082ca6347d06d1851b47d805374eced8de56ee224adbafec95
+
+build:
+  skip: true  # [win or py<39]
+  number: {{ build }}
+
+  # add build string so packages can depend on mpi variants explicitly:
+  # `yac * mpi_mpich_*` for mpich
+  # `yac * mpi_openmpi_*` for openmpi
+  # `yac * mpi_*` for any mpi
+
+  {% set mpi = mpi or "mpich" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: "{{ mpi_prefix }}_py{{ CONDA_PY }}_h{{ PKG_HASH }}_{{ build }}"
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ name }} * {{ mpi_prefix }}_py{{ CONDA_PY }}*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - autoconf
+    - automake
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - {{ mpi }}
+    - "{{ mpi }}-mpifort"
+    - yaxt =*=mpi_{{ mpi }}_*
+    - liblapack
+    - libopenblas =*=*openmp*  # [osx]
+    - libfyaml
+    - python
+    - pip
+    - setuptools
+    - cython
+    - mpi4py
+    - numpy
+  run:
+    - {{ mpi }}
+    - "{{ mpi }}-mpifort"
+    - yaxt =*=mpi_{{ mpi }}_*
+    - libopenblas =*=*openmp*  # [osx]
+    - libfyaml
+    - python
+    - mpi4py
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - yac
+  requires:
+    - pip
+  commands:
+    - pip check
+    - test -f ${PREFIX}/lib/libyac_core.a
+    - test -f ${PREFIX}/include/yac.h
+    - test -f ${PREFIX}/include/yac_core.h
+    - test -f ${PREFIX}/include/yac.mod
+    - test -f ${PREFIX}/include/yac_core.mod
```

Comment:
Since we talked about tests and you figured out how to build Python bindings: it should be easy to use Python bindings to create a small but non-trivial test.